### PR TITLE
feat: add readme to repo

### PR DESCRIPTION
**Pull request by mihai-sysbio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#### Data files for [MetabolicAtlas.org](https://metabolicatlas.org)
+
+This repository centralizes data hosted in other repositories to facilitate the deployment of Metabolic Atlas. The long-term vision is to automate the deployment process, which would circumvent the need to duplicate these respositories.
```
Comment (e0) [inline README.md:3]: It's not clear how automating the deployment process leads to not needing to duplicate the repos. Perhaps we could add some more details about that, or remove this sentence for now.

**Review by e0 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mihai-sysbio) [reply README.md]: I've added a few more words there, is the message clearer now?

Comment (e0) [reply README.md]: It helps, but does that mean this `data-files` repo would become unnecessary when the automation is in place?

Comment (mihai-sysbio) [reply README.md]: Maybe, it's hard to say prior to the implementation of this automation.

Comment (e0) [reply README.md]: Ok, I personally would choose to either expand even more about it or remove the mention of automating the deploy. But it's a minor issue and I think it's good to let people know that we are aware of the duplication at least.

**Pull request merged**